docs/clustering/logical-replication-setup.rst

```diff
@@ -0,0 +1,147 @@
+        --publish 5434:5432 \
+        crate/crate:nightly-4.8.0-20220401 \
+        -Cdiscovery.type=single-node
+
+We apply an offset to the ports of each cluster, so they don't overlap. The
+Admin UIs of ``cluster-publishing`` and ``cluster-subscribing`` are accessible at
+http://localhost:4201/ and http://localhost:4202/ respectively. All SQL
+statements discussed below can be executed via the corresponding Admin UI.
+
+As the subscriber pulls changes from the publisher, the publisher needs to accept
+incoming network connections from the subscriber. In our example setup, this is
+automatically given.
+
+Setting up a publication
+========================
+
+.. NOTE::
+
+  All SQL statements in this section are executed on ``cluster-publishing``.
+
+Before setting up the replication, we create a simple table that is going to be
+the subject of replication. A table with the same name must not exist yet on
+``cluster-subscribing``.
+
+.. code-block:: sql
+
+  CREATE TABLE doc.temperature_data (
+    ts TIMESTAMP NOT NULL,
+    temperature FLOAT NOT NULL
+  );
+
+Next, a publication is created. The publication marks our table as being
+available for replication, but otherwise does not imply any activity yet.
+
+.. code-block:: sql
+
+  CREATE PUBLICATION temperature_publication FOR TABLE doc.temperature_data;
+
+To verify the publication was created successfully, we query the
+``pg_publication`` system table. It should contain one row with the publication
+just added.
+
+.. code-block:: sql
+
+  SELECT *
+  FROM pg_publication;
+
+
+With this, we are already all set on the publication side.
+
+Setting up a subscription
+=========================
+
+.. NOTE::
+
+  All SQL statements in this section are executed on ``cluster-subscribing``.
+
+A subscription consists of connection information to the publishing cluster as
+well as the name of the previously created publication to subscribe to.
+
+Using the ``mode`` parameter, the connection can be established via either the
+transport protocol or the PostgreSQL protocol. By setting the parameter to
+``pg_tunnel``, we use the PostgreSQL protocol.
+
+For a full overview of available parameters, please see the CREATE SUBSCRIPTION
+reference.
+
+.. code-block:: sql
+
+  CREATE SUBSCRIPTION temperature_subscription
+  CONNECTION 'crate://host.docker.internal:5433?user=crate&mode=pg_tunnel'
+  PUBLICATION temperature_publication;
+
+.. NOTE::
+
+  The ``password`` parameter is omitted, as local connections using the ``crate``
+  user don't require one. If you are connecting remotely, provide the credentials
+  of a user with ``DQL``, ``DML``, and ``DDL`` privileges on published tables.
+
+After a few seconds, the table ``doc.temperature_data`` should appear on
+``cluster-subscribing``. At this point, it is still empty as we didn't insert
+any data yet.
+
+To verify the operational status of the subscription, the system tables
+pg_subscription and pg_subscription_rel can be queried. The below query returns
```

Comment:
```suggestion
``pg_subscription`` and ``pg_subscription_rel`` can be queried. The below query returns
```

Reply:
Those are going to become links to the reference documentation, but it's not possible yet, as cross-references are resolved against `latest` and those tables are only available in `master`. There are a few other missing links as well, hence this is still a draft PR.